app/extensions/brave/content/scripts/blockFlash.js

```diff
@@ -4,6 +4,18 @@
 
 const adobeRegex =
   new RegExp('//(get\\.adobe\\.com/([a-z_-]+/)*flashplayer|www\\.macromedia\\.com/go/getflash|www\\.adobe\\.com/go/getflash|helpx\\.adobe\\.com/flash-player/([a-z_-]+/)*flash-player)', 'i')
+const exemptHostRegex =
+  new RegExp('(\\.adobe\\.com|www\\.google(\\.\\w+){1,2}|^duckduckgo\\.com|^search\\.yahoo\\.com)$')
+
+function shouldIntercept(location) {
+  if (!location)
+    return true;
```

Comment:
[minor] please use curly braces around the function body and no semicolon

this would normally be caught by `npm run lint` but the linter excludes content scripts :(